src/rachanaGJun20/ExtractIteamId.java

```diff
@@ -0,0 +1,51 @@
+/*Assignment 33 : 22nd July 2020
+
+Extract ItemId from below input
+
+String input = https://www.facebook.com/technocredits/itemId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+
+Output : 
+item id is 12345 
+        OR
+no item id present.*/
+package rachanaGJun20;
+
+public class ExtractIteamId {
+
+	void extractItemIdFromString(String str) {
+		System.out.println("Input: "+str);
+		if (str.contains("itemId")) {
+			String[] str1 = str.split("/");
+			int strindex = str1.length;
+			String str2 = str1[strindex - 1];
+			if (str2.contains("&")) {
+				String[] str3 = str2.split("&");
+				for (int index = 0; index < str3.length; index++) {
+					if (str3[index].contains("itemId=")) {
+					int id = Integer.parseInt(str3[index].replace("itemId=", "").trim());
+						System.out.println("Output: Itemid is : " + id);
+					}
+				}
```

Comment:
if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}

Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}

This two line in try block and it will work.